src/rpc/quorums.cpp

```diff
@@ -1036,6 +1036,60 @@ static UniValue submitchainlock(const JSONRPCRequest& request)
     return true;
 }
 
+static void dkginfo_help(const JSONRPCRequest& request)
+{
+    RPCHelpMan{"dkginfo",
+               "Return information regarding DKGs.\n"
+               "Enabled only for Masternode and works only when SPORK_17_QUORUM_DKG_ENABLED spork is ON.\n",
+               {
+                       {},
+               },
+               RPCResults{},
+               RPCExamples{""},
+    }.Check(request);
+}
+
+static UniValue dkginfo(const JSONRPCRequest& request)
+{
+    dkginfo_help(request);
+
+    if (!fMasternodeMode) {
+        throw JSONRPCError(RPC_INVALID_REQUEST, "RPC allowed only for Masternodes");
+    }
+
+    const NodeContext& node = EnsureAnyNodeContext(request.context);
+    const ChainstateManager& chainman = EnsureChainman(node);
+    const LLMQContext& llmq_ctx = EnsureLLMQContext(node);
+
+    llmq::CDKGDebugStatus status;
+    llmq_ctx.dkg_debugman->GetLocalDebugStatus(status);
+    UniValue ret(UniValue::VOBJ);
+    ret.pushKV("nActiveDKGs", (int)status.sessions.size());
```

Comment:
please avoid c-style casts
```suggestion
    ret.pushKV("nActiveDKGs", int(status.sessions.size()));
```
or if it compiles
```suggestion
    ret.pushKV("nActiveDKGs", int{status.sessions.size()});
```